src/lib.rs

```diff
@@ -594,6 +608,13 @@ impl VadConfig {
 #[cfg(test)]
 mod tests {
     use super::*;
+    use anyhow::Result;
+    use hound::WavReader;
```

Comment:
I was just using this fully qualified name in the other tests here so should change those to just be `WavReader` just to clean things up a bit

Reply:
Also not sure why the new imports below this are needed except for the fs and path imports.

Reply:
Tried another way and later find it complicated things again, deleted the code but forget to delete the imports.